scn/post_upload_s3.bash

```diff
@@ -68,24 +68,38 @@ do
 		#move to HOLD location
 		
 		if [ ! `aws s3 ls s3://${S3HOLD}/${ulidFromJson}/`]; then    #this check is different than normal post_upload, we don't use the extra folder level
-			aws s3 cp --recursive ${DEPOSIT}/${ulidFolder}/${ulidFolder}/ s3://${S3HOLD}/${ulidFromJson}/ #this does not copy empty folders from DEPOSIT as folders do not actually exist in s3
+			packageName="package_$ulidFolder"
+			packageExt="zip"
+
+			cd ${DEPOSIT}/${ulidFolder}/
+
+			#It would be awesome to someday zip everything while it is being streamed.
+			echo "beginning zip of ${DEPOSIT}/${ulidFolder}/${ulidFolder}/"
+			zip -r $packageName ${ulidFolder}/ #There are two layers of ${ulidFolder}
+
+			shasum ${packageName}.${packageExt} > ${packageName}.sha
+
+			echo "test: ${DEPOSIT}/${ulidFolder}/$packageName"
+			aws s3 cp ${packageName}.${packageExt} s3://${S3HOLD}/${ulidFromJson}/
+			aws s3 cp ${packageName}.sha s3://${S3HOLD}/${ulidFromJson}/
```

Comment:
I decided to go with saving a .sha file because it seemed to have value incase the data needs to be accessed outside dataverse. Dataverse on its end expects the file to be there.